### DISCUSSION TITLE
MSIX Installer requires elevated permission

### DISCUSSION THREAD

**Discussion by zaochih** [microsoft/winget-pkgs#175902] [Q&A]
Dear community:

I'm writing a manifest whose MSIX package requires administrator permission. I've tried adding `ElevationRequirement: elevationRequired` to the manifest, but it seems doesn't work.

![](https://github.com/user-attachments/assets/ed914851-e8ef-4562-9b4e-dfe5e691ea7b)

Here's the manifest: https://github.com/LiCaoZ/winget-pkgs/tree/master/manifests/m/Microsoft/PCManager/3.14.6.0

Any suggestion is appreciated! :)

**Comment by Kirill638849776** [comment]
Hi! The problem is that the `ElevationRequirement` field in the WinGet manifest is not intended to request administrator rights when installing MSIX packages. It is used only to specify the elevation requirements for the installer itself (for example, for `.exe` installers).

Elevation of rights works differently for MSIX packages:

1. MSIX packages **can be installed without administrator rights** if they are signed and configured accordingly.
2. If your MSIX package **actually needs** administrator rights, this should be specified in the MSIX manifest itself (in the `AppxManifest.xml` file) in the `Capabilities` section.

In your case, try the following:

1. **Make sure** that the WinGet manifest correctly specifies `InstallerType: msix`.
2. **Check** that the MSIX package itself is signed with a trusted certificate (this is important for installation without admin rights).
3. If the installation still requires permissions, **try adding** the `InstallerSwitches` field with the value `/forceappinstall` or another key supported by your installer.

Example:
```yaml
InstallerSwitches:
  Silent: /quiet
  SilentWithProgress: /passive
